packages/cdk-routing-split/src/add/handler.ts

```diff
@@ -0,0 +1,25 @@
+import { APIGatewayEvent, Context } from 'aws-lambda';
+import { createLogger, Logger, serializeError } from '@daysmart/aws-lambda-logger';
+import { action } from './action';
+import { HttpError } from '../http-error';
+
+export const handler = async (event: APIGatewayEvent, context: Context): Promise<any> => {
+    let logger!: Logger;
```

Comment:
Nathan found aws-lambda types for node so we're trying them out. get-origin doesn't use APIGateway so i left it out. It might be a cloudfront event? W're not sure yet.